vendor/wp-now/src/symlink-manager.ts

```diff
@@ -0,0 +1,220 @@
+import { PHP, UnmountFunction } from '@php-wasm/universal';
+import fs from 'node:fs/promises';
+import { FileChangeInfo } from 'node:fs/promises';
```

Comment:
Nitpick: These two imports can be merged.

```suggestion
import fs, { FileChangeInfo } from 'node:fs/promises';
```